### ISSUE TITLE
When ``'pointsize'`` is not ``10pt`` Japanese ``'manual'`` badly renders in PDF

### ISSUE THREAD

**Issue by jfbu** [sphinx-doc/sphinx#3394]
Subject: when ``'pointsize'`` is used, the paper dimensions get wrong with ``'manual'`` class.

Test file (I picked up some random Japanese text...)

```
Welcome to FOO's documentation!
===============================

.. toctree::
   :maxdepth: 2


CHAPTER 1
---------

A

TeXが未定義の命令名を発見しました．おそらく入力の誤りでしょう．もしこ
のエラーがLaTeX命令の処理中に生じた場合は，その命令は間違った位置に置か
れています．例えば，リスト環境の中でないのに\item命令が使われた場合など
です．また，``\documentclass`` 命令がない場合にもこのエラーが生じます．


CHAPTER 2
---------

B
```

with 
```
latex_elements = {
    # The paper size ('letterpaper' or 'a4paper').
    #
    # 'papersize': 'letterpaper',

    # The font size ('10pt', '11pt' or '12pt').
    #
    'pointsize': '21pt',
}
```
and ``'manual'`` docclass and ``'ja' `` language.

### Problem
- The output is bad:

![capture d ecran 2017-02-05 a 17 04 22](https://cloud.githubusercontent.com/assets/2589111/22627577/f3937e92-ebc5-11e6-93b4-8ead229042f5.png)

The page dimensions are much bigger than the expected letterpaper.

#### Explanation of the problem

This is due to incompatibility of ``jsbook`` document class with ``geometry`` package if no special measures are taken.

#### Expected result

Something like:

![capture d ecran 2017-02-05 a 17 05 37](https://cloud.githubusercontent.com/assets/2589111/22627599/2b549492-ebc6-11e6-9833-7eb90fcd1e18.png)

In this screen shot, Adobe Reader said page dimensions are those of US letter, as expected.

### Environment info
- Sphinx version: 1.5.2
- TeX version: TeXLive 2016, more or less up-to-date.

This will be fixed at #3387 

Related: #3356

**Comment by jfbu** [comment]
(the issue was illustrated with `21pt` for drama, but it is also there for `12pt`)

**Comment by jfbu** [comment]
fixed at 09324bd and parent commits (#3387)
